doc/rst/tools/mason/mason.rst

```diff
@@ -411,113 +608,122 @@ Mason uses the ``Spack`` package specification.
           mvapich2, built with pgi compiler, with support for multiple fabrics
 
 
-So, to install ``HDF5`` version 1.10.1 built with gcc including a high level interface and without
-MPI, the following spec expression would be used::
+Resuming the example, the result of the install given ``openssl`` as the sole argument would output the following::
+
+  $ mason external install openssl
+  ==> Installing zlib
+  ==> Fetching http://zlib.net/fossils/zlib-1.2.11.tar.gz
+  ==> Staging archive: /$HOME/.mason/spack/var/spack/stage/zlib-1.2.11-cpdvq4e7otjepbwdtxmgk5bzszze27fj/zlib-1.2.11.tar.gz
+  ==> Created stage in /$HOME/.mason/spack/var/spack/stage/zlib-1.2.11-cpdvq4e7otjepbwdtxmgk5bzszze27fj
+  ==> No patches needed for zlib
+  ==> Building zlib [Package]
+  ==> Executing phase: 'install'
+  ==> Successfully installed zlib
+  Fetch: 4.84s.  Build: 4.24s.  Total: 9.08s.
+  
+  ==> Installing openssl
+  ==> Fetching http://www.openssl.org/source/openssl-1.0.2k.tar.gz
+  ==> Staging archive: /$HOME/.mason/spack/var/spack/stage/openssl-1.0.2k-fwnsee6qcvbbgvmgp3f5hio6dwg6nh2d/openssl-1.0.2k.tar.gz
+  ==> Created stage in /$HOME/.mason/spack/var/spack/stage/openssl-1.0.2k-fwnsee6qcvbbgvmgp3f5hio6dwg6nh2d
+  ==> No patches needed for openssl
+  ==> Building openssl [Package]
+  ==> Executing phase: 'install'
+  ==> Successfully installed openssl
+  Fetch: 3.37s.  Build: 3m 11.76s.  Total: 3m 15.13s.
+  ######################################################################## 100.0%
+  ######################################################################## 100.0%
+
+As you can see, Mason not only goes and gets the package specified, but also all of the dependencies
+of the package specified. Packages are installed into their own directories down paths that make it
```

Comment:
> Packages are installed into their own directories down paths that make it impossible for two packages have namespaces that collide.

Packages are installed into unique directories such that it is impossible for package namespaces to collide.